packages/cli/src/cli.ts

```diff
@@ -331,8 +331,16 @@ async function getPaymentKey(network: CLINetworkAdapter, args: string[]): Promis
  */
 async function getStacksWalletKey(network: CLINetworkAdapter, args: string[]): Promise<string> {
   const mnemonic = await getBackupPhrase(args[0]);
+  const derivationPathPrompt = await prompt([
+    {
+      type: 'input',
+      name: 'derivationPath',
+      message: 'Enter custom derivation path or press enter to use the default: ',
```

Comment:
The default path should be shown in the prompt

Reply:
Done